lib/fastlane/plugin/revenuecat_internal/helper/github_helper.rb

```diff
@@ -16,9 +16,11 @@ def self.get_pr_resp_items_for_sha(sha, github_token, rate_limit_sleep, repo_nam
           sleep(rate_limit_sleep)
         end
 
+        current_branch = Actions.git_branch
```

Comment:
Hmm this trusts that we are in the correct branch before this. I wonder if we should be passing the base branch as a parameter instead to make it more explicit... This probably works for now so we can change it in the future if we have to.

Reply:
OHHHHH, I totally meant to make that change but did this to get things to work 🙈 Thanks for catching!